README.yaml

```
@@ -76,108 +79,117 @@ usage: |-
     provider "aws" {
       region = var.region
     }
-
+    
     module "vpc" {
-      source     = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=tags/0.8.0"
-      namespace  = var.namespace
-      stage      = var.stage
-      name       = var.name
+      source     = "cloudposse/vpc/aws"
+      version    = "0.28.1"
```

This is what we do in other module README example snippets: https://github.com/cloudposse/terraform-aws-vpc#examples

```suggestion
      # Cloud Posse recommends pinning every module to a specific version
      # version = "x.x.x"
```